molecule.yml

```diff
@@ -0,0 +1,19 @@
+---
+ansible:
```

Comment:
I think that we may need to trim down the sequence of commands to run during a test because the `check` command is failing:

```yaml
molecule:
  test:
    sequence:
      - destroy
      - syntax
      - create
      - converge
      - idempotence
      - verify
```

Failure output of `check` for `xenial64`, although this occurs with `trusty64` too:

```
--> Starting Ansible Run ...
Idempotence test passed.
--> Performing a "Dry Run" of playbook ...

PLAY [all] *********************************************************************

TASK [Check Ubuntu release] ****************************************************
skipping: [ansible-ntp]

TASK [debug] *******************************************************************
fatal: [ansible-ntp]: FAILED! => {"failed": true, "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'dict object' has no attribute 'stdout'\n\nThe error appears to have been in '/Users/hcastro/Projects/ansible-ntp/playbook.yml': line 15, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - debug: msg=\"Running Ubuntu version {{ ubuntu_release.stdout|float }}\"\n      ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes.  Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

NO MORE HOSTS LEFT *************************************************************
 [WARNING]: Could not create retry file 'playbook.retry'.         [Errno 2] No
such file or directory: ''


PLAY RECAP *********************************************************************
ansible-ntp                : ok=0    changed=0    unreachable=0    failed=1
```

I suspect that there is some incompatibility with `check` and the `raw` module.